pom.xml

```diff
@@ -141,7 +141,7 @@
     <codahale.metrics.version>3.1.5</codahale.metrics.version>
     <avro.version>1.7.7</avro.version>
     <avro.mapred.classifier>hadoop2</avro.mapred.classifier>
-    <jets3t.version>0.9.4</jets3t.version>
+    <javax.activation.version>1.1.1</javax.activation.version>
```

Comment:
This is fine; if it's a version only ever used once, I also find it OK to inline to avoid one level of indirection.

Reply:
I'll inline. It's not like its one of those dependencies that anyone doing their own build will ever want to change.